src/Samples/TestStackdriver.cs

```diff
@@ -28,7 +28,8 @@ internal class TestStackdriver
             VideoSizeViewName,
             "processed video size over time",
             VideoSize,
-            Distribution.Create(BucketBoundaries.Create(new List<double>() { 0.0, 16.0 * MiB, 256.0 * MiB })),
+            Sum.Create(),
+            //Distribution.Create(BucketBoundaries.Create(new List<double>() { 0.0, 16.0 * MiB, 256.0 * MiB })),
```

Comment:
Why is this commented?

Reply:
I used a different aggregation for testing. Removing the comment.